### DISCUSSION TITLE
Tooltip on hovering the nodes and connections

### DISCUSSION THREAD

**Discussion by amitjoy** [eckig/graph-editor#100] [Q&A] [answered]
Is there any way to show tooltips on hovering the nodes/connections/connectors?

**Comment by eckig** [comment, marked as answer]
Have you read up on how to [add custom skins](https://github.com/eckig/graph-editor/wiki/3.-Adding-Custom-Skins)?

With your own skin added to the editor you can add any JavaFX element you like to the node and modify it in any way, like adding listeners.
